### DISCUSSION TITLE
TypeError : Cannot read properties of undefined (reading 'speed')

### DISCUSSION THREAD

**Discussion by chonlatit-tpit** [nolimits4web/swiper#7384] [Q&A]
Hi,

We use swiper in our project and we occasionally found the error from error monitoring tools (datadog).
The error is come form /swiper/esm/components/core/slide/slideNext.js (see attached files).

We try to reproduced the bug but cant. We used to have the same bug in other parts of our code and we solve this with

```
speed = this?.params?.speed
```

In this case, Is it possible to create pull requests for fixed this one.

Our project used

nextjs (v.12.2)
swiper (v.6.8.4)

<img width="1007" alt="Screen Shot 2023-10-09 at 11 39 25 AM" src="https://github.com/nolimits4web/swiper/assets/79088614/fd92fe4d-2ef4-4918-aef0-b3124f0114b0">

**Comment by ramsaysewell** [comment]
I'm having a similar issue. Did this ever get resolved for you @chonlatit-tpit ?

**Reply by chonlatit-tpit** [reply]
@ramsaysewell yep, it's fix on v.11.2.0

https://github.com/nolimits4web/swiper/commit/8c6a3c6f63915db82415e2d7829ece6c624ace1e

**Reply by ramsaysewell** [reply]
thanks, I'll make sure my project is using that version 🎉
